backend/src/pong/game/game.service.ts

```diff
@@ -1,7 +1,7 @@
 import { Injectable } from '@nestjs/common';
 import { InjectRepository } from '@nestjs/typeorm';
 import { AbstractService } from '../../shared/abstract.service';
-import { Repository } from 'typeorm';
+import { InsertResult, Repository } from 'typeorm';
```

Comment:
This import sneaked back in hehe

Reply:
pffffff
this is a "vscode is too slow" thing for me.. I open the file, it's all the same colour, I close the file....
Now I opened the file, waited 3 seconds, and the `InsertResult` became a different colour......